comm/src/main/scala/coop/rchain/comm/transport/GrpcTransportReceiver.scala

```diff
@@ -144,7 +137,7 @@ object GrpcTransportReceiver {
                   else
                     Metrics[F].incrementCounter("stream.chunks.dropped") >>
                       Log[F].debug(msgDropped) >>
-                      msg.path.deleteSingleFile[F] >>
+                      Sync[F].delay(cache.remove(msg.key)) >>
```

Comment:
I just wanted to say that I think the old team used file cache instead of memory cache is because rchain used to take storing-big-files as a sell point(which I think dappy wanted to do at very first) and using file cache can make the node not to blow up easily.

Reply:
I agree, but this kind of optimization should not be part of transport layer. The idea is to refactor it in way that we can add e.g. LMDB cache if needed for large files. This PR is intermediate solution fix the leak and to enable transition to better solution.

Reply:
I agree with. Just want to mention that.